scss/_base_code.scss

```diff
@@ -20,10 +21,10 @@
   }
 
   pre {
-    background: $color-x-dark;
-    border: 0;
-    border-radius: .5rem;
-    color: $color-mid-dark;
+    background-color: $color-light;
+    border: 1px solid $color-mid-dark;
+    border-radius: 2px;
```

Comment:
@barrymcgee in my opinion, the border-radius of a component is a direct design attribute. Therefore should not change based on the font-size. A design attribute is like a signal of or style. If the style changes it no longer becomes instantly recognisable as "Vanilla".

Let defer this conversation to dev catch up if you don't mind.